static/app/components/versionHoverCard.tsx

```diff
@@ -236,10 +236,8 @@ const StyledVersion = styled(Version)`
   max-width: 190px;
 `;
 
-const ClipboardIconWrapper = styled('span')`
-  &:hover {
-    cursor: pointer;
-  }
+const StyledCopyToClipboardButton = styled(CopyToClipboardButton)`
+  margin-block: -${space(1)};
```

Comment:
damn why do we need it 😢

Reply:
it gets taller with the real button in there:

Remove negative margin:
<img width="297" alt="VersionHoverCard - without negative margin" src="https://user-images.githubusercontent.com/187460/236585253-9b167a72-6042-466f-8fe7-5c42f258ee18.png">

Existing Prod:
<img width="298" alt="VersionHoverCard - prod" src="https://user-images.githubusercontent.com/187460/236585256-4f0e284f-ef76-4afc-bd66-de19cd24026b.png">

Reply:
there is extra space left/right of the button, but i left that alone.

Reply:
i forgot about `size='zero'`, less background on hover to see, but it keeps space here better without the margin.

updated the screens in the description with that.